main.tf

```diff
@@ -276,32 +280,87 @@ locals {
     space.name => space.space_id
   }
 
+  # Helper for property resolution with fallback to defaults
+  stack_property_resolver = {
+    for stack in local.stacks : stack => {
+      # Simple property resolution with fallback
+      administrative                   = coalesce(try(local.stack_configs[stack].administrative, null), var.administrative)
+      autoretry                        = try(local.stack_configs[stack].autoretry, var.autoretry)
+      additional_project_globs         = try(local.stack_configs[stack].additional_project_globs, var.additional_project_globs)
+      autodeploy                       = coalesce(try(local.stack_configs[stack].autodeploy, null), var.autodeploy)
```

Comment:
Huh. Why does it do a coalesce + try on some, but not all... these should be `bool` types, right? I imagine we could remove the `coalesce` on.